app/modules/sre/webhook_helper.py

```diff
@@ -320,7 +320,7 @@ def toggle_webhook(ack, body, logger, client):
     channel = hook["channel"]["S"]
 
     webhooks.toggle_webhook(id)
-    message = f"Webhook {name} has been {'enabled' if hook['active']['BOOL'] else 'disabled'} by <@{username}>"
+    message = f"Webhook {name} has been {'disabled' if hook['active']['BOOL'] else 'enabled'} by <@{username}>"
```

Comment:
Minor, but won't this now log the wrong state since it will return `disabled` if `hook['active']['BOOL'] == True`?

Reply:
Ah, nvm, I see what's going on and this is because it's the toggle function.